README.md

```diff
@@ -211,6 +211,7 @@ List keys:
 * [vco-plugins](http://kokkinizita.linuxaudio.org/linuxaudio/index.html) - LADSPA plugin sporting anti-aliased oscillators ([◼](https://packages.debian.org/sid/vco-plugins)) `▒`
 * [x42-plugins](https://github.com/x42/x42-plugins) - Collection of LV2 plugins ([◼](https://packages.debian.org/sid/x42-plugins)) `▒`
 * [zam-plugins](http://www.zamaudio.com/) - Plugins for high quality audio processing ([◼](https://packages.debian.org/sid/zam-plugins)) `▒`
+* [ZL Audio Plugins](https://zl-audio.github.io/) - ZL Audio makes audio plugins and audio processing tools
```

Comment:
Thanks.

I'd rather link to https://github.com/ZL-Audio since it's the only page listing all plugins the website on github.io only lists ZLEqualizer, and the download link links to the github organization page again.

Reply:
Yeah. I changed that.